qa/pull-tester/loadbalancer.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+import sys
+
+workers = int(sys.argv[1])
+selectedChunk = int(sys.argv[2])        # bash style, they go from 1 to workers
+
+# Restrict selectedChunk value in [1,workers] (no silent fail)
+selectedChunk = min(selectedChunk, workers)
+selectedChunk = max(selectedChunk, 1)
+
+## STEP-BY-STEP example
+## sys.argv:
+## ['prova1.py', '4', '1', 'paymentdisclosure.py,63', 'txn_doublespend.py,15', 'txn_doublespend.py', '--mineblock,16', 'getchaintips.py,48', 'rawtransactions.py,101']
```

Comment:
Possibly `'prova1.py'` in this comment should be updated

Reply:
Nice! Yes, corrected with the actual filename